package/MDAnalysis/core/topologyattrs.py

```diff
@@ -409,6 +411,9 @@ def getattr__(atomgroup, name):
             raise AttributeError("'{0}' object has no attribute '{1}'".format(
                     atomgroup.__class__.__name__, name))
 
+    @deprecate(message="Instant selector AtomGroup['<name>'] or AtomGroup.<name> "
+               "is deprecated and will be removed in 1.0. "
+               "Use AtomGroup.select_atoms('name <name>') instead.")
     def _get_named_atom(group, name):
```

Comment:
I adopted `_get_named_atom` to also print the requested name. That way I found out we apparently use it for more then just atom names

```python
from MDAnalysisTests import make_Universe
import warnings
warnings.simplefilter('always')
req_attrs = {'resnames': 'UNK',
             'resids': 1,
             'names': 'X',
             'tempfactors': 0.0,
             }
attrs = list(req_attrs.keys())
u = make_Universe(attrs, trajectory=True)
u.atoms.write('test.crd')
```

output

```bash
/home/max/foss/molecular-dynamics/mdanalysis/mdanalysis/package/MDAnalysis/core/groups.py:1539: DeprecationWarning: `_get_named_atom` is deprecated!
Instant selector AtomGroup['<name>'] or AtomGroup.<name> is deprecated and will be removed in 1.0. Use AtomGroup.select_atoms('name <name>') instead.
  return self._get_named_atom(attr)
chainIDs
/home/max/foss/molecular-dynamics/mdanalysis/mdanalysis/package/MDAnalysis/core/groups.py:1539: DeprecationWarning: `_get_named_atom` is deprecated!
Instant selector AtomGroup['<name>'] or AtomGroup.<name> is deprecated and will be removed in 1.0. Use AtomGroup.select_atoms('name <name>') instead.
  return self._get_named_atom(attr)
segids
/home/max/foss/molecular-dynamics/mdanalysis/mdanalysis/package/MDAnalysis/coordinates/CRD.py:229: UserWarning: Supplied AtomGroup was missing the following attributes: tempfactors. These will be written with default values. 
  "".format(miss=', '.join(missing_topology)))
```

This is running with numpy 1.12.1